engine/build.gradle

```diff
@@ -2,16 +2,11 @@ apply from: '../config/gradle/common.gradle'
 apply plugin: 'java-library'
 [compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
 
-import org.reflections.Reflections;
-import org.reflections.scanners.SubTypesScanner;
-import org.reflections.scanners.TypeAnnotationsScanner;
-import org.reflections.util.ConfigurationBuilder;
-import org.reflections.util.ClasspathHelper;
-
 // Dependencies needed for what our Gradle scripts themselves use. It cannot be included via an external Gradle file :-(
 buildscript {
     repositories {
         mavenCentral()
+        mavenLocal()
```

Comment:
Whilst this is very useful for local development, it often causes problems too. I've had issues with unpublished older snapshots of a library in the `mavenLocal` repository superceeding published newer ones before, so I wouldn't recommend including it long-term.